operators/vision/vision.hpp

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "ocos.h"
+#include "decode_image.hpp"
+#include "encode_image.hpp"
+
+FxLoadCustomOpFactory LoadCustomOpClasses_PPP_Vision =
```

Comment:
With these new operators, can we remove the previous operators checked in from super-resolution model?

Reply:
Yes. Rachel has been updating an example app that currently uses them. Once we switch that over I will remove them.